hbase-shell/src/main/ruby/hbase_constants.rb

```diff
@@ -32,75 +32,79 @@
 java_import('java.lang.Boolean') { |_package, name| "J#{name}" }
 
 module HBaseConstants
+  ALLOW_PARTIAL_RESULTS = 'ALLOW_PARTIAL_RESULTS'.freeze
```

Comment:
Please do this sorting in a different jira

Reply:
Yeah, I also got confused at first. If overall sorting change was reasonably smaller to identify, it would have been better to include as part of this PR.

Reply:
Really? well, okay.

Reply:
HBASE-24843 / #2236